src/hotspot/share/code/codeBlob.hpp

```diff
@@ -155,17 +155,17 @@ class CodeBlob {
   virtual void purge(bool free_code_cache_data, bool unregister_nmethod);
 
   // Typing
-  bool is_nmethod() const                     { return _kind == Blob_Nmethod; }
-  bool is_buffer_blob() const                 { return _kind == Blob_Buffer; }
-  bool is_runtime_stub() const                { return _kind == Blob_Runtime_Stub; }
-  bool is_deoptimization_stub() const         { return _kind == Blob_Deoptimization; }
-  bool is_uncommon_trap_stub() const          { return _kind == Blob_Uncommon_Trap; }
-  bool is_exception_stub() const              { return _kind == Blob_Exception; }
-  bool is_safepoint_stub() const              { return _kind == Blob_Safepoint; }
-  bool is_adapter_blob() const                { return _kind == Blob_Adapter; }
-  bool is_vtable_blob() const                 { return _kind == Blob_Vtable; }
-  bool is_method_handles_adapter_blob() const { return _kind == Blob_MH_Adapter; }
-  bool is_upcall_stub() const                 { return _kind == Blob_Upcall; }
+  bool is_nmethod() const                     { return _kind == CodeBlobKind::Blob_Nmethod; }
+  bool is_buffer_blob() const                 { return _kind == CodeBlobKind::Blob_Buffer; }
+  bool is_runtime_stub() const                { return _kind == CodeBlobKind::Blob_Runtime_Stub; }
+  bool is_deoptimization_stub() const         { return _kind == CodeBlobKind::Blob_Deoptimization; }
+  bool is_uncommon_trap_stub() const          { return _kind == CodeBlobKind::Blob_Uncommon_Trap; }
+  bool is_exception_stub() const              { return _kind == CodeBlobKind::Blob_Exception; }
+  bool is_safepoint_stub() const              { return _kind == CodeBlobKind::Blob_Safepoint; }
+  bool is_adapter_blob() const                { return _kind == CodeBlobKind::Blob_Adapter; }
+  bool is_vtable_blob() const                 { return _kind == CodeBlobKind::Blob_Vtable; }
+  bool is_method_handles_adapter_blob() const { return _kind == CodeBlobKind::Blob_MH_Adapter; }
+  bool is_upcall_stub() const                 { return _kind == CodeBlobKind::Blob_Upcall; }
```

Comment:
The `Blob_` prefix is now redundant since we always have to prefix with CodeBlobKind::. Just a suggestion if you want to shorten these.

Reply:
Good suggestion